frontend/src/Commons/CustomLink/CustomLink.style.ts

```diff
@@ -10,18 +11,21 @@ const StyledLinkContainer = styled.div<StyledLinkProps>`
   display: flex;
   align-items: center;
   justify-content: center;
-  height: ${(props) => (props.size === 'sm' ? '27px' : props.size === 'md' ? '25px' : '40px')};
-  width: ${(props) => (props.size === 'sm' ? '52px' : props.size === 'md' ? '200px' : '300px')};
+  height: 100%;
+  width: 100%;
+  max-height: 42px;
+  max-width: ${(props) => getButtonSize(props.size, false)};
```

Comment:
Thanks for fixing my mistake :grin:

Reply:
Totally no problem! Actually one thing I can add here is a constant for isDesktop and isMobile. For clarity, thanks for making me realize by commenting!